core/src/main/java/hudson/model/Queue.java

```diff
@@ -397,7 +396,7 @@ public void load() {
                     while ((line = in.readLine()) != null) {
                         AbstractProject j = Jenkins.get().getItemByFullName(line, AbstractProject.class);
                         if (j != null)
-                            j.scheduleBuild();
+                            j.scheduleBuild(new Cause.UserIdCause());
```

Comment:
Does not look like a good cause.
The original code would use the `LegacyCodeCause` while here we replace it with User Cause which is not true. 

TBH I think it is time to just remove the "old queue format" support. This format was already replaced in 2008 when I started using Hudson, and there is no sense to keep this code at all. First relevant diff in Git, did not dive into the SVN history https://github.com/jenkinsci/jenkins/commit/8bb5e7f5b45b7f2994137f74ac87cb3c20388cbb. Any concerns @jenkinsci/core?

Reply:
We probably should drop it. Perhaps we revert this change and make a follow up change with its removal.